python/templates/macros/collections.jinja2

```diff
@@ -46,7 +46,7 @@ std::vector<{{ member.full_type }}> {{ class.bare_type }}Collection::{{ member.n
 {% set real_index = start_index + index %}
   for (auto& obj : entries) {
     if (obj->m_{{ relation.name }}) {
-      m_refCollections[{{ real_index }}]->emplace_back(obj->m_{{ relation.name }}->getObjectID());
+       m_refCollections[{{ real_index }}]->emplace_back(obj->m_{{ relation.name }}->getObjectID());
     } else {
        m_refCollections[{{ real_index }}]->push_back({podio::ObjectID::invalid, 0});
```

Comment:
```suggestion
      m_refCollections[{{ real_index }}]->emplace_back(obj->m_{{ relation.name }}->getObjectID());
    } else {
      m_refCollections[{{ real_index }}]->push_back({podio::ObjectID::invalid, 0});
```
two spaces instead of three